store/index.js

```diff
@@ -91,11 +93,11 @@ export default function configureStore(initialState, persistorStorage = null) {
                     observable.subscribe({
                         next: (args) => {
                             if(args.key && args.key.indexOf(KEY_PREFIX) === 0){
-                              var keyspace = args.key.substr(KEY_PREFIX.length)
+                                var keyspace = args.key.substr(KEY_PREFIX.length);
```

Comment:
nothing major but can we change this to const?

Reply:
Sure!